tests/test_bytes_utils.py

```diff
@@ -20,3 +20,8 @@ def test_split_and_join(in_str, in_bytes):
     res_str, res_bytes = bu.split(bu.join(in_str, in_bytes))
     assert res_str == in_str
     assert res_bytes == in_bytes
+
+
+def test_int_to_bytes_length():
```

Comment:
This name is not too descriptive. Maybe something like
```suggestion
def test_int_to_bytes_raises_when_input_is_too_big():
```
would be better? (Cf. [earlier comment](https://github.com/vil02/puzzle_generator/pull/72#discussion_r1705043047)).

Why not to place this test _closer_ to `test_int_to_bytes`?

Reply:
Wouldn't it would be too verbose?

Reply:
> Wouldn't it would be too verbose?

No.